docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1329,26 +1334,29 @@ function getRemoteMediaSettings() {
   if (meeting && meeting.mediaProperties.remoteVideoTrack) {
     const videoSettings = meeting.mediaProperties.remoteVideoTrack.getSettings();
     const {frameRate, height} = videoSettings;
-
     remoteVideoResElm.innerText = `${height}p ${Math.round(frameRate)}fps`;
   }
 }
 
-let resolutionInterval;
+let localVideoResolutionInterval;
+let remoteVideoResolutionInterval;
 const INTERVAL_TIME = 3000;
 
-function meetingsResolutionCheckInterval() {
-  resolutionInterval = setInterval(() => {
+function localVideoResolutionCheckInterval() {
+  localVideoResolutionInterval = setInterval(() => {
```

Comment:
I think localVideoResolutionCheckInterval() can be called multiple times (it's called from getUserMedia() which is called for example every time user changes the camera), so we need to clear the previous interval before setting the new one

Reply:
Yes, added a check to clear interval if exist before starting a new one for both `localVideoResolutionCheckInterval` and `remoteVideoResolutionCheckInterval `

Reply:
thanks, but I can see that neither `localVideoResolutionInterval` nor `remoteVideoResolutionInterval` is cleared when  `clearVideoResolutionCheckInterval()` is called, so we may end up calling `clearInterval` on an interval that's already cleared. Not sure if this is big problem, I guess `clearInterval()` will just not do anything in that case? it would be cleaner to reset these variables to undefined when the interval is cleared.

Reply:
Yes, that's correct. `clearInterval` just clears the interval associated with that particular id returned by setInterval. Also there's nothing clearInterval will do if interval is already cleared but yeah just for code cleanliness we can set those variables to null.

Reply:
Updated the code